src/eherkenning/backends.py

```diff
@@ -14,6 +14,14 @@ class eHerkenningBackend(_eHerkenningBackend):
     Custom backend to identify users based on the KvK number instead of RSIN
     """
 
+    def _sync_eherkenning_session_context(self, request, user):
+        session_context = EHerkenningSessionContext(request)
+        session_context.persist_eherkenning_state_for_user(
+            user=user,
+            is_branch_restricted=bool(user.vestiging),
```

Comment:
For the `EHerkenningOIDCBackend` you have `is_branch_restricted=user.vestiging != ""`. Should be consistent.